src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) {

```

Comment:
The method parameter 'filePath' should be checked to ensure that it is not null or empty before attempting to write to the file. This can prevent potential `NullPointerException` or unexpected behavior if an invalid path is passed.